modules/javafx.controls/src/main/java/javafx/scene/control/skin/VirtualFlow.java

```diff
@@ -1800,6 +1800,16 @@ public T getCell(int index) {
         return accumCell;
     }
 
+    /**
+     * The amount of pixels by which to adjust the position when the the scroll bar track is clicked.
```

Comment:
should it be the height of the viewport, so to speak?

Reply:
Yes, that would also be my favorite choice for the click on the scrollbar track as well as for the PageUp/PageDown.
But for reasons of backwards compatibility, I didn't want to change it here.